SteamKit2/SteamKit2/Networking/Steam3/CMServerList.cs

```diff
@@ -0,0 +1,334 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SteamKit2.Networking.Steam3
```

Comment:
One last bit that's been bothering me. Do we want to introduce this namespace? I've always viewed the `Networking` subdirectory as something for the internal bits for connections, whereas managing a server list is something that lives on top of `CMClient`.